test/langtools/tools/javac/7142086/T7142086.java

```diff
@@ -24,11 +24,19 @@
 /*
  * @test
  * @bug 7142086
```

Comment:
I would add the current bug number `8337334` after the previous one

Reply:
Thanks for your advice, the bug number `8337334 ` has been added.